.circleci/workflows.yml

```diff
@@ -48,12 +52,25 @@ parameters:
   run-container-test:
     type: boolean
     default: false
+  prune:
+    type: boolean
+    default: false
+    description: "Do you want to [docker system prune -f --all] ? (clean up all cached docker images)"
+  tag_latest:
+    type: boolean
+    default: false
+    description: "Is this latest version of the Product ?"
+  tag_latest_support:
+    type: boolean
+    default: true
+    description: "Is this a latest support version of the Product ? (if so minor version tagging docker images). True by default."
+    
 orbs:
   gravitee: gravitee-io/gravitee@1.0
   keeper: gravitee-io/keeper@0.6.2
   slack: circleci/slack@4.8.1
   snyk: snyk/snyk@1.1.2
-  am: gravitee-io/gravitee-am@1.0.8
+  am: gravitee-io/gravitee-am@1.0
```

Comment:
What's the reason of the downgrade ? 🤔

Reply:
it's not a downgrade, when this `@1.0` is   set on the gravitee orb version, it means it's the latest version